coordinator/src/routes/dispention/release.ts

```diff
@@ -29,26 +30,36 @@ export default [
   {
     method: "POST",
     path: `${basePath}/Task/$release`,
-    handler: taskValidatorHandler(
-      async (bundle: fhir.Parameters, request: Hapi.Request, responseToolkit: Hapi.ResponseToolkit) => {
-        request.logger.info("Sandbox release response")
-        const messageId = "d5a20db9-6d76-4aeb-a190-9a85843b01bf"
-        const sandboxResponse: fhir.Bundle = {
-          resourceType: "Bundle",
-          type: "searchset",
-          total: 2,
-          id: messageId,
-          identifier: {
-            value: messageId
-          },
-          entry: [
-            {resource: genericBundle1, fullUrl: `urn:uuid:${bundle1Id}`},
-            {resource: genericBundle2, fullUrl: `urn:uuid:${bundle2Id}`}
-          ]
-          //TODO: find reasonable task response
-        }
-        return responseToolkit.response(sandboxResponse)
+    handler: async (request: Hapi.Request, responseToolkit: Hapi.ResponseToolkit) => {
+      const fhirValidatorResponse = await externalFHIRValidation(request)
+      if (fhirValidatorResponse.issue.length > 0) {
+        return responseToolkit.response(fhirValidatorResponse).code(400)
       }
-    )
+
+      const requestPayload = getPayload(request) as fhir.Resource
+
+      if (requestPayload.resourceType !== "Parameters") {
```

Comment:
Should we do the same thing for `$process-message` to check the message is a `Bundle`?